packages/components/src/navigation/style.scss

```diff
@@ -0,0 +1,34 @@
+.components-navigation {
+	width: 272px;
+	padding: 35px 30px;
+	background: #191e23;
+	color: #ccd0d4;
+}
+
+.components-navigation-back {
```

Comment:
Seeing some minor differences between this component and the design's spacing/sizing.

<img width="141" alt="Screen Shot 2020-07-22 at 1 15 52 PM" src="https://user-images.githubusercontent.com/10561050/88164927-896d9700-cc1d-11ea-90d7-6662baf96ea0.png">
<img width="116" alt="Screen Shot 2020-07-22 at 1 15 48 PM" src="https://user-images.githubusercontent.com/10561050/88164933-8a062d80-cc1d-11ea-8beb-d7831d9cd93f.png">

Reply:
I just removed my less than ideal CSS and left it as a primary button so it can get the attention it needs. I also remembered this Figma is the design source not necessarily the prototype, https://www.figma.com/file/ft4SsfQd8L3WiqF3qStUty/WooCommerce-Navigation?node-id=1%3A13133

![Screen Shot 2020-07-23 at 1 31 33 PM](https://user-images.githubusercontent.com/1922453/88244918-5c86b600-cce9-11ea-93e7-be269db175e4.png)